shared/app.js

```diff
@@ -70,6 +73,8 @@ module.exports = Backbone.Model.extend({
      * Initialize the `ClientRouter` on the client-side.
      */
     if (!isServer) {
+      var ClientRouter = this.options.ClientRouter || require(defaultRouterModule);
```

Comment:
@saponifi3d  we have this PR https://github.com/rendrjs/rendr-examples/pull/15 to make it work with Rendr `1.0.1`